src/test/resources/log4j2-test.xml

```diff
@@ -1,16 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<Configuration status="DEBUG">
+<Configuration status="error">
   <Appenders>
     <Console name="Console" target="SYSTEM_OUT">
       <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
     </Console>
   </Appenders>
   <Loggers>
     <Logger name="com.iopipe" level="debug">
-      <AppenderRef ref="Console"/>
     </Logger>
     <Root level="error">
-      <AppenderRef ref="Console"/>
+      <AppenderRef ref="Console" />
```

Comment:
Do you mean to add this extra space here?

Reply:
The space is not needed at all, I did intentionally add the space since it makes XML a bit easier to read for me at least.